### DISCUSSION TITLE
G6 has been "Hunting Transmitter" for 4 hours

### DISCUSSION THREAD

**Discussion by tkjones** [NightscoutFoundation/xDrip#3417] [Q&A] [answered]
Hi all,
My G6 transmitter lost connection in the middle of the night. It's been at "Hunting Transmitter" for 4 hours. The sensor is 2.5 days old and the transmitter is 82 days old. I've forgotten the device, restarted collector, removed all Dexcom devices from my bluetooth list and rebooted my phone. It just doesn't seem to want to reconnect. Should I change the transmitter? The battery is showing Voltage A at 309 and Voltage B at 290.
Thanks in advance for any help!

**Comment by Navid200** [comment]
Post a screenshot of the status page.

**Reply by gitanO0** [reply]
I've had this sort of thing happen twice in the past few weeks. A reboot and doing the "change the last character of the transmitter ID routine" fixed the issue. 

But it's odd. I've not had this issue at all and then twice in a few weeks? Android Pixel 7 Pro.

**Comment by tkjones** [comment]
![20240402_062210_xDrip](https://github.com/NightscoutFoundation/xDrip/assets/10778155/35187037-e277-4b41-9a50-4386ce629811)

**Reply by tkjones** [reply, marked as answer]
I tried changing the last digit of the transmitter id and then switching back but that didn't work. I ended up swapping out the transmitter and it connected almost immediately. I'm now waiting for the sensor to warm up.

**Reply by gitanO0** [reply]
Ok. Odd.

**Reply by tkjones** [reply]
I think the transmitter died. Even though the battery was still fine, it stopped transmitting. The expiry on the box is 2024-05-17 but they probably should last well beyond that date.

**Reply by tkjones** [reply]
New transmitter is working well. I'll mark that as the answer.

**Reply by gitanO0** [reply]
Sure would love to know what was up with that transmitter.
